code/game/machinery/computer/buildandrepair.dm

```diff
@@ -28,7 +28,7 @@
 					to_chat(user, "[WT] must be on to complete this task.")
 					return
 					
-				playsound(loc, 'sound/items/Welder.ogg', 25, 1)
+				playsound(loc, 'sound/items/welder.ogg', 25, 1)
```

Comment:
not sure why you renamed this, the file is still called Welder/Welder2.ogg, if you were trying to rename this file lowercase it didnt work.

Reply:
Fixed.